events/events.go

```diff
@@ -112,11 +113,15 @@ func writeEvent(job *engine.Job, event *utils.JSONMessage, eventFilters filters.
 			if v == field {
 				return false
 			}
+			if partialMatch && strings.Contains(field, v) {
```

Comment:
short version is not partial match. Short version would be using `strings.HasPrefix` instead.